Advice

Cvaguila opened the pull request.

acodeinprogress commented:
Hi @Cvaguila,

Congratulations on making your first pull request!

There are a few changes that you'll need to make here before we can get this merged in:

1. I see that you have a cindy-v-advice.html folder with a file called your-name-advice.html inside. Instead, you'll want to rename that file to something like cindy-v-advice.html and place it in the advice folder. Once you do that, you can delete teh cindy-v-advice folder that is in the root folder of the repo.
2. It appears as though you overwrote the style.css file. Instead, you'll want to create your own stylesheet and add your styles in there. You'll also need to restore the style.css file back to its previous state. You can see the differences in the file by clicking [here](https://github.com/skillcrush/105-practice/pull/1622/files).

Once you make those changes, you'll need to run `git add -A`, then commit and push. Your updated code will join your pull request here and we can get that merged in.

xoxo,
Sharon

Cvaguila commented:
Thank you, I’ll work on what needs to be fixed.

Best, 
Cindy  

Sent from my iPhone

> On Oct 13, 2020, at 1:11 PM, Sharon <notifications@github.com> wrote:
> 
> ﻿
> Hi @Cvaguila,
> 
> Congratulations on making your first pull request!
> 
> There are a few changes that you'll need to make here before we can get this merged in:
> 
> I see that you have a cindy-v-advice.html folder with a file called your-name-advice.html inside. Instead, you'll want to rename that file to something like cindy-v-advice.html and place it in the advice folder. Once you do that, you can delete teh cindy-v-advice folder that is in the root folder of the repo.
> It appears as though you overwrote the style.css file. Instead, you'll want to create your own stylesheet and add your styles in there. You'll also need to restore the style.css file back to its previous state. You can see the differences in the file by clicking here.
> Once you make those changes, you'll need to run git add -A, then commit and push. Your updated code will join your pull request here and we can get that merged in.
> 
> xoxo,
> Sharon
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or unsubscribe.

Cvaguila commented:
Hi- I went ahead and renamed my fold and added it to the advice folder. How would I go about deleting the cindy-v-advice folder from the root folder in the repo?

Thank you, Cindy

redrambles commented:
Hey @Cvaguila !

Ok - we are getting closer! A few things remain.

1) We have a `normalize.css` file in the root of the folder which we don't need - so please delete it. You can delete it directly from inside the Terminal, or you can just open up the folder and drag the file to the Trash.

2) You have made changes to the `css/style.css` file - but this is a 'default' stylesheet that many students are using, and we don't want to edit it. I see that you have created your own custom stylesheet, which is great - but we'll want to return that default css/style.css file back to it's original value. You can do that by running:
`git checkout master style.css`

This will replace your advice branch's version of the style.css file with the (unchanged) version on your master branch.

3) Finally - you'll want to rename your custom stylesheet so that it doesn't have a space in the name. Spaces in paths can break things on servers, so we don't want them in our file names. Once you have made this change, you will need to edit your link to your stylesheet from inside your advice page so that the two connect. You will also want to make your path relative, rather than absolute, as it is now. This means your path to css file should be relative to your advice file. Something like this:
`../css/your-style-sheet-with-no-spaces.css`. The relative path will work no matter where the project is hosted - whereas an absolute path will only work on your computer  -and only if you don't move your project folder around.

Once you have made these changes, please add with `git add -A`, commit as per usual and then push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Cheers,
Ann :)

redrambles commented:
Hello @Cvaguila !

Ok - so while still in your advice branch, run the following:
`git checkout master css/style.css`

This will replace your current version (which has been modified) of the 'css/style.css' file in your advice branch with a copy of the version of the same file on your master branch, which is untouched. 

Does this help?

Cvaguila commented:
Hi,
Thank you, I totally understand.

For some reason I can't get back on my advice branch. I've tried everything
I know, as you can see from my screen shot. Please let me know if i
missed anything that would bring me back onto my "advice branch"

Cindy Aguila

On Tue, Oct 20, 2020 at 1:51 PM Ann Cascarano <notifications@github.com>
wrote:

> Hello @Cvaguila <https://github.com/Cvaguila> !
>
> Ok - so while still in your advice branch, run the following:
> git checkout master css/style.css
>
> This will replace your current version (which has been modified) of the
> 'css/style.css' file in your advice branch with a copy of the version of
> the same file on your master branch, which is untouched.
>
> Does this help?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/1622#issuecomment-713132076>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQXKDP4K6ILH7TVI2PMFXDTSLXZ5BANCNFSM4SKJEJOQ>
> .
>

Cvaguila commented:
Hi,

I have a quick question for you. I have been trying to get back on to my
advice branch and I can only pull up what you can see on my screen shot.
As you can see it shows it on my "master branch, but then it shows
"project-105/advice" meaning I'm in the folder, right? How do I get back to
my advice branch?
I have tried "git checkout <branch name> and it says
Cindy@cindy-surface MINGW64 ~/105-practice (master)
$ git checkout advice
error: The following untracked working tree files would be overwritten by
checkout:
        advice/cindy-v-advice.html
Please move or remove them before you switch branches.
Aborting

What does this mean?

Please help!

Cindy Aguila



<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Oct 20, 2020 at 3:27 PM cynthia verdugo-aguila <
cverdugoaguila@gmail.com> wrote:

> Hi,
> Thank you, I totally understand.
>
> For some reason I can't get back on my advice branch. I've tried
> everything I know, as you can see from my screen shot. Please let me know
> if i missed anything that would bring me back onto my "advice branch"
>
> Cindy Aguila
>
> On Tue, Oct 20, 2020 at 1:51 PM Ann Cascarano <notifications@github.com>
> wrote:
>
>> Hello @Cvaguila <https://github.com/Cvaguila> !
>>
>> Ok - so while still in your advice branch, run the following:
>> git checkout master css/style.css
>>
>> This will replace your current version (which has been modified) of the
>> 'css/style.css' file in your advice branch with a copy of the version of
>> the same file on your master branch, which is untouched.
>>
>> Does this help?
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skillcrush/105-practice/pull/1622#issuecomment-713132076>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQXKDP4K6ILH7TVI2PMFXDTSLXZ5BANCNFSM4SKJEJOQ>
>> .
>>
>

redrambles commented:
Hey @Cvaguila !

This means that you are not currently in your advice branch, and that you have a pending change that would be overwritten if you change branches right now. This is what I suggest:

1) Run `git add -A` and then `git stash`. This will 'tuck away' this changed file for now and we can retrieve it later if we want to.
2) Run `git checkout advice` to move to your advice branch
3) Now you can go ahead and follow the instructions we wrote in our previous reply. :)

​Keep us posted on your progress!			

Cheers,
Ann :)

acodeinprogress commented:
Hey @Cvaguila,

We haven't heard from you in a while so I'm going to close this pull request. Feel free to submit a new one when you're ready!

xoxo,
Sharon

Cvaguila commented:
Hi Skillcrush,

Please don't close it. I made a draft email. And i forgot to send it to
you, but here it is.


I went ahead and did what you said and I was continuing with the
instructions for the previous email and an error message came up.
Please see the screenshot that I have attached.

Cindy Aguila



<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#m_-7983373879384590491_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Oct 27, 2020 at 5:41 PM Ann Cascarano <notifications@github.com>
wrote:

> Hey @Cvaguila <https://github.com/Cvaguila> !
>
> This means that you are not currently in your advice branch, and that you
> have a pending change that would be overwritten if you change branches
> right now. This is what I suggest:
>
>    1. Run git add -A and then git stash. This will 'tuck away' this
>    changed file for now and we can retrieve it later if we want to.
>    2. Run git checkout advice to move to your advice branch
>    3. Now you can go ahead and follow the instructions we wrote in our
>    previous reply. :)
>
> ​Keep us posted on your progress!
>
> Cheers,
> Ann :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/skillcrush/105-practice/pull/1622#issuecomment-717624334>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQXKDPZNG7RZUUERQUAJQWDSM5SDNANCNFSM4SKJEJOQ>
> .
>

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Cvaguila commented:
Hi- Since you closed this pull request, I am stuck and I'm not sure where
to start from. Please advise.

Thank you!
Cindy Aguila



On Wed, Nov 11, 2020 at 5:41 PM cynthia verdugo-aguila <
cverdugoaguila@gmail.com> wrote:

> Hi Skillcrush,
>
> Please don't close it. I made a draft email. And i forgot to send it to
> you, but here it is.
>
>
> I went ahead and did what you said and I was continuing with the
> instructions for the previous email and an error message came up.
> Please see the screenshot that I have attached.
>
> Cindy Aguila
>
>
>
>
> <http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Virus-free.
> www.avg.com
> <http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> <#m_-4884495394726665010_m_-7983373879384590491_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>
> On Tue, Oct 27, 2020 at 5:41 PM Ann Cascarano <notifications@github.com>
> wrote:
>
>> Hey @Cvaguila <https://github.com/Cvaguila> !
>>
>> This means that you are not currently in your advice branch, and that you
>> have a pending change that would be overwritten if you change branches
>> right now. This is what I suggest:
>>
>>    1. Run git add -A and then git stash. This will 'tuck away' this
>>    changed file for now and we can retrieve it later if we want to.
>>    2. Run git checkout advice to move to your advice branch
>>    3. Now you can go ahead and follow the instructions we wrote in our
>>    previous reply. :)
>>
>> ​Keep us posted on your progress!
>>
>> Cheers,
>> Ann :)
>>
>> —
>> You are receiving this because you were mentioned.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/skillcrush/105-practice/pull/1622#issuecomment-717624334>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AQXKDPZNG7RZUUERQUAJQWDSM5SDNANCNFSM4SKJEJOQ>
>> .
>>
>
>
> <http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail> Virus-free.
> www.avg.com
> <http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
> <#m_-4884495394726665010_DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
>
